open-metadata-resources/open-metadata-deployment/charts/odpi-egeria-lab/values.yaml

```diff
@@ -44,7 +44,7 @@ imageDefaults:
   registry: docker.io
   namespace: odpi
   tag: latest
-  pullPolicy: Always
+  pullPolicy: IfNotPresent
```

Comment:
For *master* I set this value to Always intentionally, since the master code is always changing, and otherwise the image gets cached.
If that's inappropriate you can override.
It gets set to 'IfNotPresent' when the version number is locked prior to release

Reply:
See https://github.com/odpi/egeria/issues/4046 also for the reference to dockerhub limits. You can authenticate to dockerhub to increase the limits, or wait a while. I think this is clearer than accidentally using invalid code.